definitions/managed sevices.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Managed sevices
+status: Feedback Appreciated
+category: property
+---
+## Managed sevices
+### What it is
+ A complete safety net to your technical environment. It is the trust that you can have in a team to look after, both proactively and reactively, an ever-changing database environment. Above all, it is the knowledge that you have a team of experts to cover all areas of issues!
+
+Listed below are the main challenges Managed Services addresses and resolves.
+
+### Problem it addresses
+
+Cybersecurity:
+The information security battle is ongoing. There is a constant need to evaluate your infrastructure for vulnerabilities and to assess your defenses for weaknesses.
+
+Cloud:
+Many businesses turn to cloud to ease the burden of their IT, but cloud doesn’t mean there isn’t any work for your IT team. The unfamiliar environments can be challenging for teams to support.
+
+Physical and virtual machine support:
+Whether your data center uses physical servers or you’ve virtualized everything, there’s work to be done every day to provision new servers, deploy patches, and monitor how well things are running. Desktop devices also need that kind of attention
+
+Vendor management:
+Facilitating and maintaining  relationships between your organization and vendors/partners, negotiating contracts, creating standards for the vendors, and finding the best available vendors,is too tedious of a job .
+
+But Managed Sevices eases everything . Lets find out how! 
+
+
+
+
+### How it helps
+  1.Managed services can make sure your firewalls are configured properly, administer identity and access management solutions, and verify the security of your data in the cloud.
```

Comment:
We don't want bullet points or numbering. This needs to be one or two paragraphs and ideally around 3-4 sentences.

Reply:
Read